src/core/generator/generator.rs

```diff
@@ -153,12 +169,17 @@ mod test {
     #[derive(Deserialize)]
     struct JsonFixture {
         url: String,
-        body: serde_json::Value,
+        response: serde_json::Value,
     }
 
     fn parse_json(path: &str) -> JsonFixture {
         let content = std::fs::read_to_string(path).unwrap();
-        serde_json::from_str(&content).unwrap()
+        let json_content: serde_json::Value = serde_json::from_str(&content).unwrap();
+
+        JsonFixture {
+            url: json_content["request"]["url"].as_str().unwrap().to_string(),
+            response: json_content["response"]["body"].clone(),
+        }
```

Comment:
- Lets make this function async.
- Use serde to deserialize instead of hand building JsonFixture.
- Drop the unwraps.

Reply:
done